### ISSUE TITLE
AQ20 Ossirian does too little damage.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#1091]
Shamelessly stolen from #1061.

Example log showing an average of under 2k auto-attacks from Ossirian on Darrowshire (obvious he is stuck in supreme mode from the warstomp damage).
legacy-logs

Textual references from 2005/6.

    http://web.archive.org/web/20061109203510/http://www.wow-tactics.de/20_taktiken/aq20/ossirian.php
    http://www.tentonhammer.com/guides/tentonhammers-guide-to-the-ruins-of-ahn-qiraj-section-2-6
    https://www.tacticalgamer.com/forum/mass-multiplayer/world-of-warcraft/world-of-warcraft-tactics-strategies-resources/23099-aq-20-boss-tactics
    http://olivier.rs-team.org/strats_wow/AQ20OSSIRIAN.pdf
    http://leftoversraiding.org/index.php/topic/817/1/pulling-ossirian.html#post6739

Video from 2005/6 showing damage when weakened.

    Tank PoV
    https://www.youtube.com/watch?v=Mlh8lSaBqPk
    https://www.youtube.com/watch?v=qb0zZ3xkKLk
    https://www.youtube.com/watch?v=Y2ITGk5_8hc

Basically:

    his Warstomp ability seems to be ok (~3k on legacy-logs for supreme mode vs ~1k in textual and video evidence from 2006 for weakened state)
    his auto-attack seems to be heavily undertuned even if you account for the armor mitigation differences between an appropriately geared tank for the instance as shown in the classic videos and a full T2 one as shown in recent logs.

His auto-attack should have a variance of 10.000-14.000 before armor mitigation and defensive stance modifier in supreme mode.

**Comment by QAkvastskaft** [comment]
Probably should fix #1061 first :)

**Comment by Road-block** [comment]
Bestiary is BC.

**Comment by Tobschinski** [comment]
So what...
